README.md

```diff
@@ -103,6 +103,12 @@ class UrlGenerator < Alephant::Broker::LoadStrategy::HTTP::URL
 end
 ```
 
+##### Revalidate Strategy
+
+```ruby
```

Comment:
Yup, there should (and will be) but as it's still a WIP I didn't want to write the docs, only to have to rewrite them if I change the approach. :)